db/schema.rb

```diff
@@ -190,6 +190,9 @@
     t.boolean "psychiatric_liaison"
     t.string "psychiatric_liaison_reason_not"
     t.boolean "next_hearing"
+    t.boolean "hourly_travel_costs"
```

Comment:
This doesn't belong in this PR, I don't think - it's from the travel cost migration.